File: src/funcs/FuncPack.java
package funcs;

import castle.Game;
import cells.Item;

public class FuncPack extends FuncSrc {

	public FuncPack(Game game) {
		super(game);
	}

	@Override
	public void DoFunc(String cmd) {
		game.echoln("背包中物品如下：");
		for (Item item : game.getTheItems()) {
			game.echoln(item.toString());
		}
	}

}

File: src/funcs/using/FuncPack.kt
package funcs.using

import castle.Game
import cells.Item
import funcs.FuncSrc

class FuncPack(game: Game) : FuncSrc(game) {

    override fun DoFunc(cmd: String) {
        game.echoln("背包中物品如下：")
        for (item in game.theItems) {
            game.echoln(item.toString())
        }
    }

}
